packages/core/src/match.ts

```diff
@@ -1,37 +0,0 @@
-import { State } from './State';
-import { StateValue, EventObject, MachineContext } from './types';
-
-export type ValueFromStateGetter<
-  T,
-  TContext extends MachineContext,
-  TEvent extends EventObject
-> = (state: State<TContext, TEvent>) => T;
-
-export type StatePatternTuple<
-  T,
-  TContext extends MachineContext,
-  TEvent extends EventObject
-> = [StateValue, ValueFromStateGetter<T, TContext, TEvent>];
-
-export function matchState<
```

Comment:
This was a public export so we should document its removal

Reply:
Added a changeset